Quadriga/src/main/webapp/WEB-INF/views/resolvers/edit.jsp

```diff
@@ -1,6 +1,55 @@
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
 <%@taglib uri="http://www.springframework.org/tags/form" prefix="form"%>
 
+<script>
+	
+$(function() {
+	$("#testEdit").click(function(){
+		performAction(this)
+		});
+});
+
+function handleResult(status){
+	
+	if (status == "SUCCESS") {
+	$(".input-group-addon").html('<i class="fa fa-check-circle"></i>');
+	} else if (status == "FAILURE") {
+	$(".input-group-addon").html('<i class="fa fa-times"></i>');	
+	}
+	
+}
+
+function performAction(obj) {
+	
+	var data = {};
+	data['projectName'] = $("#projectName").val();
+	data['projectDescription'] = $('#projectDescription').val();
+	data['projectUrl'] = $('#projectUrl').val();
+	data['handlePattern'] = $('#handlePattern').val();
+	data['resolvedHandlePattern'] = $('#resolvedHandlePattern').val();
+	data['handleExample'] = $('#handleExample').val();
+	data['resolvedHandleExample'] = $('#resolvedHandleExample').val();
+	
+	$
+	.ajax({
+		type : "POST",
+		url : "${pageContext.servletContext.contextPath}/auth/resolvers/testEdit",
+		data : {
+			data : JSON.stringify(data)
+		},
+		beforeSend: function() {
+			$(".input-group-addon").html('<i class="fa fa-spinner fa-spin" aria-hidden="true"></i>');
+        },
+        success : function(data,status) {
+			handleResult(data);
+		},
+		error : function(e) {
+			console.log("ERROR: ", e.responseText);
+		}
+	});
+}
+	
+</script>
```

Comment:
this is more or less exactly the same script as for adding, right? Just put that into an external js file and reuse it.